cellxgene_schema_cli/cellxgene_schema/migrate_files/automigrate_terms.json

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
@brian-mott I noticed at least one deprecated HsapDv term from the latest dry-run is missing a re-map. I don't see a re-map for HsapDv:0000090, which is contained in this dataset https://api.cellxgene.cziscience.com/curation/v1/collections/e1a9ca56-f2ee-435d-980a-4f49ab7a952b

I haven't done a full diff to see if other terms are missing

Reply:
HsapDv:0000092 is also missing a re-map, same collection

Reply:
These should all be covered in the auto migrate_terms.json line 449 with the listed donor_ids

Reply:
Ah got it. Thanks for looking into it